data/json/hobbies.json

```
@@ -91,5 +91,59 @@
     "description": "You enjoyed camping in the vast wilderness as a means of escaping the pace of modern life.  The wilderness isn't filled with people, so it must be safe.  Right?  It must be.",
     "points": 2,
     "skills": [ { "level": 2, "name": "survival" }, { "level": 2, "name": "firstaid" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "painting",
+    "name": "Painting",
+    "description": "You know your way around a canvas and a brush. Fine motor coordination and basic applied chemistry come more naturally to you.",
+    "points": 2,
+    "skills": [ { "level": 2, "name": "chemistry" }, { "level": 2, "name": "cutting" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "cinematography",
+    "name": "Cinematography",
+    "description": "For many years you have exercised your usage of the camera. You have a deeper understanding of devices and conveyances.",
+    "points": 1,
+    "skills": [ { "level": 2, "name": "driving" }, { "level": 2, "name": "traps" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "thespian",
+    "name": "Thespian",
+    "description": "Your are something of a local celebrity at the theater. Bonuses to communication.",
```

```suggestion
    "description": "Your are something of a local celebrity at the theater.",
```
I don't think it needs a particular explanation as to why it gives +speech 😛